t/t2035-checkout-locked.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
```

Comment:
For the comments below, please see also https://public-inbox.org/git/20180611203958.GA1306@Sonnenschein.localdomain/ as a reference. It includes improvements from review comments.

Reply:
Thanks